tools/StaticAnalysis/ExampleAnalyzer/utils.ps1

```diff
@@ -81,8 +81,11 @@ function Get-ExamplesDetailsFromMd {
     # Skip the 1st because it is $EXAMPLES_HEADING. Extract content without title.
     $examplesContentWithoutTitle = $examplesContent -split $SINGLE_EXAMPLE_TITLE_HEADING_REGEX | Select-Object -Skip 1
 
-
     foreach ($exampleContent in $examplesContentWithoutTitle) {
+        #skip the autogenerated example
+        if($exampleContent -match "(autogenerated)"){
```

Comment:
```suggestion
        if($exampleContent -match "autogenerated"){
```
Parenthesis "()" means group matching in regex. We can just check "autogenerated" for simplicity